chia/wallet/wallet_block_store.py

```diff
@@ -62,7 +62,12 @@ async def create(cls, db_wrapper: DBWrapper):
         await self.db.execute("CREATE INDEX IF NOT EXISTS height on block_records(height)")
 
         await self.db.execute("CREATE INDEX IF NOT EXISTS hh on block_records(header_hash)")
-        await self.db.execute("CREATE INDEX IF NOT EXISTS peak on block_records(is_peak)")
+
+        # this index was initially created and deployed without where condition "is_peak = 1"
+        # to replace the old index with the new one, uncomment the drop
+        # await self.db.execute("DROP INDEX IF EXISTS peak on block_records(is_peak)")
```

Comment:
same comment applies to this index. I think we could just drop it immediately

Reply:
Regarding the how to drop indexes.
Did you check my suggested way to [drop indexes](https://github.com/Chia-Network/chia-blockchain/pull/9297/commits/520874105269f12ffbda365a7df5dc7d92d0010f) in my other PR #9297?

Instead of only using `DROP INDEX IF EXISTS peak`, I'd check if we are going to drop the index on the correct table and dynamically building the stmts.
Imho this would be a more robust and safe way to drop indexes.

Reply:
it seems we're about to drop support for this wallet, in favor of the new. light-weight wallet.

Reply:
nice, so I can play around with some new SQLs 🥳 
are the sources of the new light wallet already published?